example/lib/presentation/samples/bar/bar_chart_sample5.dart

```diff
@@ -197,6 +204,8 @@ class BarChartSample5State extends State<BarChartSample5> {
               value1 + value2 + value3,
               value1 + value2 + value3 + value4,
               AppColors.contentColorBlue,
+                        '5',
+              const TextStyle(),
```

Comment:
Why do we have an empty `TextStyle()` here?
Because, as I see it is nullable, so we should be able to send null here (or just not specifying it)